yubihsm/backends/usb.py

```diff
@@ -26,10 +26,11 @@
 class UsbBackend(YhsmBackend):
     """A backend for communicating with a YubiHSM directly over USB."""
 
-    def __init__(self, serial: Optional[int] = None):
```

Comment:
This changes the default timeout when calling the constructor directly from 300 seconds to `None` (aka "forever"?). I'd prefer if the default value in the constructor was instead changed to 600 (since 300 isn't long enough for some operations), and then __init__.py doesn't need any changes.

Reply:
Happy to do that, although I went with this patch initially to mirror `HttpBackend`. `HttpBackend.__init__()` defaults to `timeout = None` and then in `yubihsm/backends/__init__.py` it is set to `timeout=(10, 600)`.

If you still prefer `UsbBackend.__init__(..., timeout = 600)` without any changes to `yubihsm/backends/__init__.py`, let me know and I'll update the PR. Thanks!

Reply:
Good point, I wasn't considering the HttpBackend. In that case I'm happy to merge it as-is and we can look over the defaults for both of these before release.

Reply:
I'll try to explain my reasoning in the PR description next time :smiley:

Thanks!